recipe/Makefile.conda.PAR

```diff
@@ -1,65 +1,179 @@
+# based on Makefile.inc.generic, MUMPS 5.7.3
+
+#  This file is part of MUMPS 5.7.3, released
+#  on Mon Jul 15 11:44:21 UTC 2024
+#
+################################################################################
+#
+#   Makefile.inc.generic
 #
-#  This file is part of MUMPS 5.1.2, released
-#  on Mon Oct  2 07:37:01 UTC 2017
+#   This defines some parameters dependent on your platform; you should
+#   look for the approriate file in the directory ./Make.inc/ and copy it
+#   into a file called Makefile.inc. For example, from the MUMPS root
+#   directory, use 
+#   "cp Make.inc/Makefile.inc.generic ./Makefile.inc"
+#   (see the main README file for details)
 #
-# These settings for a PC under Debian/linux with standard packages :
-# metis (parmetis), scotch (ptscotch), openmpi, gfortran
+#   If you do not find any suitable Makefile in Makefile.inc, use this file:
+#   "cp Make.inc/Makefile.inc.generic ./Makefile.inc" and modify it according
+#   to the comments given below. If you manage to build MUMPS on a new platform,
+#   and think that this could be useful to others, you may want to send us
+#   the corresponding Makefile.inc file.
```

Comment:
Comments are preserved from Makefile.inc.generic for easier diffs when mumps publishes releases.